### DISCUSSION TITLE
Does Intercepting Routes work with Static Exports?

### DISCUSSION THREAD

**Discussion by k35o** [vercel/next.js#70509] [Help]
### Summary

`Intercepting Routes` is not included in [Unsupported Features](https://nextjs.org/docs/app/building-your-application/deploying/static-exports#unsupported-features) that `Static Exports`, but does not work.

In Example, `npx next` intercepts client transitions, but `npx next build && npx serve out` does'nt.

### Example

https://stackblitz.com/edit/github-xgazpu

**Comment by icyJoseph** [comment]
Related issue, without solution, https://github.com/vercel/next.js/issues/58115

**Reply by k35o** [reply]
Does that mean `Intercepting Routes` can't work without server?

**Reply by icyJoseph** [reply]
I mean, looks like it hasn't worked, one way or another for a while. However I'd want to understand exactly what's different, like, is the router intercept a server side concept purely? Perhaps. However, a client side router should be able to do this, maybe it's just not fully implemented?
